R/get_comms_doc_info.R

```diff
@@ -0,0 +1,44 @@
+#' fill comms doc
+#'
+#' Take the template word documents and save them with specific information for the upcoming workshop in the corresponding Sharepoint channel
+#'
+#' @param info information about the workshop to be inserted into the different documents
+#' should include: startdate, enddate, title, address, venue, starttime, endtime, slug.
+#' Taken from Holy Excel Sheet
```

Comment:
```suggestion
#' @param info information about the workshop to be inserted into the different documents
```
`@param` tag can only be one line.